### DISCUSSION TITLE
[Bug] [WinUI3] Margin cropping image

### DISCUSSION THREAD

**Discussion by limeniye** [microsoft/microsoft-ui-xaml#8754] [General]
<h4>Steps</h4>

Create WinUI sample with the following code:
```XML
    <Grid Height="70" Width="70">
        <Border Background="Red"/>
        <Image Stretch="Uniform"
               Source="Assets/new.svg" Margin="20"/>
    </Grid>
```
<h4>Expectations</h3>

Image should change his size from 70/70 to 30/30 because of ```Margin="20" ```
<h4>Result</h4>

Image cropping:
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/69314237/c4233d2f-d262-4d03-a683-c5e8c74e387d)

<h4>Option info</h4>

Microsoft.WindowsAppSDK Version="1.3.230602002"
Microsoft.Windows.SDK.BuildTools Version="10.0.22621.755"
Visual Studio Community 2022 17.7.0 Preview 5.0

<h4>new.svg</h4>

![new](https://github.com/microsoft/microsoft-ui-xaml/assets/69314237/6b84b712-f9ec-4da6-aa62-fed021928de3)

**Comment by agneszitte** [comment]
@limeniye I think this could be a workaround for your specific case here using `RasterizePixelHeight` and `RasterizePixelWidth` with `SvgImageSource`. Your current SVG here has a size of 48*48, so you can turn it into a bitmap of 48x48 using the `RasterizePixelHeight` and `RasterizePixelWidth` values and then the image scales that to 30x30 with a Margin of 20.

```
	<Grid Height="70"
		  Width="70">
		<Border Background="Red" />
		<Image Margin="20"
			   Stretch="Uniform"
			   VerticalAlignment="Center">
			<Image.Source>
				<SvgImageSource UriSource="Assets/new.svg"
								RasterizePixelHeight="48"
								RasterizePixelWidth="48" />
			</Image.Source>
		</Image>
	</Grid>
```

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/16295702/bc6fda3e-3034-4259-942f-9a528c99e6b2)

(I have also answered a similar discussion you have opened on the **[Uno Platform](https://github.com/unoplatform/uno)** repo here: https://github.com/unoplatform/uno/issues/13262#issuecomment-1730218300 with another solution.)

**Reply by limeniye** [reply]
@agneszitte, thank you very much for the detailed answer.
I really appreciate your time
